Readme.md

```diff
@@ -1,4 +1,5 @@
 # Translating Typora
+This is Czech translation.
```

Comment:
You don't need to submit this `Readme.md`, since it will also be merged into this repo

Reply:
How long it will take until you implement this pull request?

Reply:
@byDave251 could you exclude this `Readme.md` from this MR?

Reply:
Excluded